Main.java

```diff
@@ -1,84 +1,94 @@
 package tester;
 
-import java.io.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.util.Scanner;
 
-public class Main{
-	static Sys system;
-	static File file;
-	static Scanner f;
-	static Scanner in;
+import tester.models.Headmate;
+import tester.models.Sys;
 
-	public static void main(String[] args){
-		in=new Scanner(System.in);
-		try{
-			file=new File("C:\\Users\\Morgan\\Desktop\\SystemMapper\\SystemMapper\\src\\tester\\data.dat");
-			f=new Scanner(file);
+public class Main {
+	private static Scanner in;
+	private static File file;
+	private static Sys system;
 
-			if(file.length()==0) init();
-			else setup();
-			system.add(new Headmate("Tayto"));
-			//edit();
+	public static void main(String[] args) {
+		in = new Scanner(System.in);
+		file = new File("data.dat").getAbsoluteFile();
```

Comment:
we weren't sure how to automatically reach the file, that's why our file was hardcoded

Reply:
Testing locally seemed to work fine for me. You should be able to pull the branch before accepting if you'd like to test for your end